invenio_communities/templates/semantic-ui/invenio_communities/details/subcommunity/index.html

```diff
@@ -0,0 +1,33 @@
+{# -*- coding: utf-8 -*-
+
+    This file is part of Invenio.
+    Copyright (C) 2024 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% extends "invenio_communities/details/base.html" %}
+
+{% set active_community_header_menu_item= 'browse' %}
```

Comment:
Perhaps we could define a macro to render subcommunities here and use it in `invenio-app-rdm` in the `browse` page?